src/converse-service-administration.js

```diff
@@ -0,0 +1,452 @@
+                }).up();
+                
+                const callback = function () {
+                    self.render();
+                }
+
+                const errback = function () {
+                    // Nothing to Do. Do not show options
+                }
+
+                // send the stanza and register the callback functions
+                _converse.connection.sendIQ(stanza, callback, errback);
+            },
+
+            render () {
+                if (_.isNil(this.servive_administration_view)) {
+                    this.servive_administration_view = new _converse.ServiceAdministrationControlboxView();
+                    _converse.servive_administration_view = this.servive_administration_view;
+                }
+                this.servive_administration_view.render();
+                const userinfo = _converse.chatboxviews.get('controlbox').el.querySelector('#chatrooms');
+                if (userinfo !== null) {
+                    userinfo.insertAdjacentElement('beforebegin', this.servive_administration_view.el);
+                }
+                return this;
+            }
+        }); 
+
+        // View of a single service-administration-option (XEP-0133)
+        _converse.ServiceAdministrationView = Backbone.NativeView.extend({
+            tagName: 'div',
+            className: 'list-item controlbox-padded d-flex flex-row',
+            events: {
+                'click': 'showServiceAdministrationChat'
+            },
+
+            render () {
+                this.el.insertAdjacentHTML('beforeend', tpl_service_administration_option({
+                    'name': this.model.attributes.name,
+                    'service_administration_name': this.model.attributes.controlbox_view_text
+                }));
+                return this.el;
+            },
+
+            showServiceAdministrationChat (ev) {
+                ev.preventDefault();
+
+                if (_.isNil(_converse.servive_administration_chat)) {
+                    _converse.servive_administration_chat = new _converse.ServiceAdministrationChat({ model: this.model});
+                } 
+
+                _converse.servive_administration_chat.model = this.model;
+                _converse.servive_administration_chat.registerResponseHandler();
+                _converse.chatboxviews.closeAllChatBoxes();
+                const element = _converse.servive_administration_chat.render();
+                if (_.isNil(_converse.chatboxviews.el.querySelector('#sendAdminCommandId'))) {
+                    const converse_html = _converse.chatboxviews.el.querySelector('#controlbox');
+                    converse_html.insertAdjacentElement('afterend', element);
+                }
+
+                u.slideIn(this.el.parentElement);
+            }
+        });
+
+        // Model for all the service-administration-options provided (XEP-0133)
+        _converse.servive_administration_model = Backbone.Model.extend({ });
```

Comment:
Object names which can be instantiated with `new` should be in BumpyCase (with a starting capital letter), not with underscores.